app/config/frontendAppConfig.scala

```diff
@@ -80,7 +80,9 @@ trait AppConfig extends ServicesConfig {
   val portalPaymentHistoryUrl: String => String
   val portalNonHybridPreviousPaymentsUrl: String => String
   def languageMap:Map[String,Lang]
-  val routeToSwitchLanguage :String => Call
+  val routeToSwitchLanguage: String => Call
+  val host: String
```

Comment:
Could `host` be taken out of the `AppConfig` trait and instead just be a private val in `FrontendAppConfig`? Look like it's only `feedbackUrl` that needs to be accessible outside of here.